File: Application/sdk/Apf/Applications/UxApplication.php
<?php
namespace Apf\Applications;

use Closure;

class UxApplication
{

    /**
     * @param Closure $callback
     * @param string $FluentThemeMode Dark or Light
     * @param array|null $args
     */
    public function __construct(Closure $callback, string $FluentThemeMode = "Dark", array $args = NULL)
    {
    }
}

File: Apf/Platform/UxApplication.cs
#nullable enable
using Avalonia;
using Avalonia.Controls;
using Avalonia.Themes.Fluent;

namespace Apf.Platform;

public class UxApplication
{
    public UxApplication(Closure closure, string fluentThemeMode , string[]? args = null)
    {
        var builder =  AppBuilder.Configure<Application>();
            builder.UsePlatformDetect();
            
            switch (fluentThemeMode)
            {
                case "Dark":
                    builder.UseFluentTheme(mode: FluentThemeMode.Dark);
                    break;
                case "Light":
                    builder.UseFluentTheme(mode: FluentThemeMode.Light);
                    break;
                default:
                    builder.UseFluentTheme(mode: FluentThemeMode.Light);
                    break;
            }
            
            builder.StartWithClassicDesktopLifetime(desktop =>
            {
                desktop.MainWindow = (Window) closure.__invoke().ToClass();
            }, args );
    }
}